network_modeling/configs/templates/1.2/dellmellanox/common/bgp.j2

```diff
@@ -8,21 +8,21 @@ router bgp 65533 vrf Customer distance 20 70 20
 router bgp 65533 vrf default distance 20 70 20
 router bgp 65533 vrf Customer maximum-paths ibgp 32
 router bgp 65533 vrf default maximum-paths ibgp 32
-{%- for can_name, can_ip in variables.CAN_IPs.items() if "ncn-w" in can_name %}
-router bgp 65533 vrf Customer neighbor {{ can_ip }} remote-as 65536
+{%- for cmn_name, cmn_ip in variables.CMN_IPs.items() if "ncn-w" in cmn_name %}
+router bgp 65533 vrf Customer neighbor {{ cmn_ip }} remote-as 65536
```

Comment:
Not for this PR, but mentioning for a possible later improvement...
We now also have the ASNs in SLS so we could use those here instead of hardcoding 65533 and 65536.

Reply:
I've created a new ticket for this. CASMNET-1091
This should be a quick fix.